m4_sessions/1901-inning/prejudice.md

```diff
@@ -1,30 +1,37 @@
 ## Group Members:
-
-
+Tanjie, Melvin, Derek, Harper
 
 ## Name of Session: 
+Prejudice
 
 ## Session Description 
-
 Turing's mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers. In this session we will identify, examine, and discuss our prejudices, biases, and our related experiences with the aim of creating concrete actionable solution to make tech and our broader communities more (inclusive**?)
 
 ## Session Outcomes 
-
 (What will students learn/leave with?): 
+Students will leave with a concrete plan for address their own biases and those of others in the wild
+
+## Session Outline
+* Intros (5-10 mins) Let students settle in the space, an icebreaker where each student writes an interesting fact about themselves on a piece of paper and puts it in a hat. Papers are drawn by a facilitator, the fact is read aloud and the group tries to guess who submitted the fact.  Norms and expectations will then be outlined. 
+
+* Implicit Bias Tests (10 - 15 mins) Students will need to bring a computer to this session.  Facilitators will choose four implicit bias tests.  Students will be randomly assigned to take one of the four test, and time permitting, will be able to another bias test of their choosing. Students will be given a few minutes to reflect on the finding either in a journal or just to themselves.
+
+* Small Group Discussion (15 mins) Students will break into small groups determined by the randomly assigned bias test (eg, those who took the bias around disabilities test will group together, those who took the racial biases test will group, etc.
```

Comment:
This is great.  What are the discussion questions you are going to have on the board?